src/xcode/ENA/ENA/Source/Services/DSCList/DSCListProvider.swift

```diff
@@ -0,0 +1,87 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+import HealthCertificateToolkit
+
+final class DSCListProvider: DSCListProviding {
+
+	// MARK: - Init
+
+	init(
+		client: DSCListFetching,
+		store: Store,
+		interval: TimeInterval = 12 * 60 * 60  // 12 hours
+	) {
+		self.client = client
+		self.store = store
+		self.interval = interval
+		// load last knows | default DSCList data
+		self.metaData = Self.loadDSCListMetaDataIfAvailable(store: store, interval: interval)
+		self.signingCertificates = CurrentValueSubject<[DCCSigningCertificate], Never>(metaData.signingCertificate)
+
+		// trigger an update immediately
+		NotificationCenter.default.addObserver(self, selector: #selector(updateListIfNeeded), name: UIApplication.willEnterForegroundNotification, object: nil)
+		updateListIfNeeded()
+	}
+
+	deinit {
+		NotificationCenter.default.removeObserver(self, name: UIApplication.willEnterForegroundNotification, object: nil)
```

Comment:
This is no longer needed;
https://developer.apple.com/documentation/foundation/notificationcenter/1413994-removeobserver
"If your app targets iOS 9.0 and later or macOS 10.11 and later, and you used addObserver(_:selector:name:object:), you do not need to unregister the observer. If you forget or are unable to remove the observer, the system cleans up the next time it would have posted to it."

Reply:
I removed it